doc/source/development/copy_on_write.rst

```diff
@@ -1,4 +1,4 @@
-.. _copy_on_write:
+.. _copy_on_write_dev:
```

Comment:
IMO I think the information on this page is value enough to be in the user guide

Reply:
Hm not sure, I think this should be considered an implementation detail? Especially since we have to change aspects of it to better accommodate for block splitting logic

Reply:
Could these two guides be cross linked at least?

(I just generally have a preference for consolidating documentation.)

Reply:
I added a link from development to user guide. As long as we aren't done I'd like to keep casual readers away from it

Reply:
Agreed with Patrick that this are implementation details and users don't have to care about, but explain the inner details of how it works that is useful for people working on the pandas code base. 

So I would also keep this separate, but of course can always put a link from the user to here for people that want to know more.